crypto/Cargo.toml

```diff
@@ -27,6 +27,7 @@ iroha_schema = { path = "../schema" }
 derive_more = { version = "0.99.17", default-features = false, features = ["deref", "deref_mut", "display"] }
 parity-scale-codec = { version = "3.2.1", default-features = false, features = ["derive", "full"] }
 serde = { version = "1.0.151", default-features = false, features = ["derive"] }
+serde_with = {version = "2.1.0", default-features = false, features = ["macros"]}
```

Comment:
```suggestion
serde_with = { version = "2.1.0", default-features = false, features = ["macros"] }
```

Reply:
On that note, I feel we should consider adding a TOML auto-formatter to the CI as well as cargo fmt. In most cases the overhead would be minimal.

Reply:
#3282